src/openforms/submissions/api/serializers.py

```diff
@@ -626,3 +626,37 @@ def validate(self, attrs: EmailVerificationData) -> EmailVerificationData:
             )
 
         return attrs
+
+
+class VerifyEmailData(EmailVerificationData):
+    code: str
+
+
+class VerifyEmailSerializer(EmailVerificationSerializer):
+    code = serializers.CharField(
+        label=_("verification code"),
+        min_length=6,
+        max_length=6,
+        write_only=True,
+    )
+
+    class Meta(EmailVerificationSerializer.Meta):
+        fields = EmailVerificationSerializer.Meta.fields + ("code",)
```

Comment:
Don't we want to explicitly define-show the fields here?I haven't seen this before for fields that's why I am asking (another question concerning best practices)

Reply:
it's the "easiest" way for inheritance. I may regret inheriting the serializer in the future, but then I could easily just define it separately entirely :) this automatically brings along the fields from the parent serializer and keeps things in sync.